src/main/resources/database/data.sql

```diff
@@ -7,6 +7,128 @@ INSERT INTO `role` (`r_name`) VALUES ('seniorHousemaid');
+ INSERT INTO `user` (u_login, u_password) VALUES ('rostyslav@mail.com','1');
+ INSERT INTO `user` (u_login, u_password) VALUES ('myroslav@mail.com','1');
+ INSERT INTO `user` (u_login, u_password) VALUES ('vas@mail.com','1');
+
+ INSERT INTO `permission` (p_name) VALUES ('change comment');
+ INSERT INTO `permission` (p_name) VALUES ('create vacancy');
+ INSERT INTO `permission` (p_name) VALUES ('get tasks');
+ INSERT INTO `permission` (p_name) VALUES ('delegate tasks');
+
+ INSERT INTO `role_permission` (r_id, p_id) VALUES (1,1);
+ INSERT INTO `role_permission` (r_id, p_id) VALUES (1,2);
+ INSERT INTO `role_permission` (r_id, p_id) VALUES (2,2);
+ INSERT INTO `role_permission` (r_id, p_id) VALUES (3,1);
+ INSERT INTO `role_permission` (r_id, p_id) VALUES (3,2);
+ INSERT INTO `role_permission` (r_id, p_id) VALUES (1,3);
+
+INSERT INTO `user_role` (u_id, r_id) VALUES (1,1);
+INSERT INTO `user_role` (u_id, r_id) VALUES (1,2);
+INSERT INTO `user_role` (u_id, r_id) VALUES (2,3);
+INSERT INTO `user_role` (u_id, r_id) VALUES (2,5);
+
+INSERT INTO `position` (p_title) VALUES ('manager');
+INSERT INTO `position` (p_title) VALUES ('admin');
+INSERT INTO `position` (p_title) VALUES ('waiter');
+INSERT INTO `position` (p_title) VALUES ('cleaner');
+
+
+INSERT INTO `employee`(u_id, p_id, e_forename, e_surname, e_patronymic, e_phone_number, e_info) VALUES (1,1,'Vasyl', 'Vasyliv', 'Vasylovich', '4509654345', 'blablabla1');
+INSERT INTO `employee`(u_id, p_id, e_forename, e_surname, e_patronymic, e_phone_number, e_info) VALUES (2,2,'Petro', 'Petriv',  'Petrovych',  '094586403', 'blablabla2');
+
+
+INSERT INTO `schedule` (p_id, start, end) VALUES (2,  '19:30:10', '20:30:10' );
+INSERT INTO `schedule` (p_id, start, end) VALUES (1,  '21:00:30', '23:30:10' );
+INSERT INTO `schedule` (p_id, start, end) VALUES (3,  '08:55:10', '10:30:10' );
+
+INSERT INTO `work_shift` (u_id, start, end, date) VALUES (1, '23:30:10', '00:40:10', CURRENT_TIMESTAMP);
+INSERT INTO `work_shift` (u_id, start, end, date) VALUES (2, '10:30:10', '15:30:10', CURRENT_TIMESTAMP);
+INSERT INTO `work_shift` (start, end, date) VALUES ( '13:30:10', '16:30:10', CURRENT_TIMESTAMP);
+
+INSERT INTO `shift_comment` (u_id, w_shift_id, c_message, c_date) VALUES (1, 1, 'message 1', CURRENT_TIMESTAMP);
+INSERT INTO `shift_comment` (u_id, w_shift_id, c_message, c_date) VALUES (2, 1, 'message 1', CURRENT_TIMESTAMP);
+INSERT INTO `shift_comment` (u_id, w_shift_id, c_message, c_date) VALUES (3, 2, 'message 1', CURRENT_TIMESTAMP);
+
+INSERT INTO `location` (l_title, l_info) VALUES ('restaurant 1', 'restaurant in hotel');
+INSERT INTO `location` (l_title, l_info) VALUES ('restaurant 2', 'restaurant outside');
+INSERT INTO `location` (l_title, l_info) VALUES ('laundry', 'laundry');
+INSERT INTO `location` (l_title) VALUES ('room 200');
+
+INSERT INTO `task` (u_id_assignee, u_id_delegator, t_message, t_status, t_frequency, updated) VALUES (1, 2, 'clean rooms', 'OPENED', 'DAILY', CURRENT_TIMESTAMP);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 4 times. ([plsql:S1192](https://sonarcloud.io/organizations/final-project/rules#rule_key=plsql%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=83&issues=AWVY-fc22xJdnw6QkSxk&open=AWVY-fc22xJdnw6QkSxk)

[ISSUE_KEY]: AWVY-fc22xJdnw6QkSxk